lib/commanded/event/handler.ex

```diff
@@ -868,6 +872,35 @@ defmodule Commanded.Event.Handler do
     {:noreply, state}
   end
 
+  def extract_error_handler_module(opts) do
+    case Keyword.get(opts, :retry, :stop) do
+      :stop ->
+        Commanded.Event.ErrorHandler.Stop
+
+      :backoff ->
+        Commanded.Event.ErrorHandler.Backoff
+
+      :grumpy_old_bastard ->
+        Commanded.Event.ErrorHandler.GrumpyOldBastard
```

Comment:
should it be just functions thou? instead of forcing to be a module

Reply:
I like that, I'll try it.

Reply:
Done